### PR TITLE
Few more splits

### PR DESCRIPTION
**Pull request by Yotona**
Kind of ugly, data types might be wrong (particularly s32s, pointers, still figuring out some arguments), mostly fleshing out smaller functions for the context. Of course, most of them just point back to big difficult functions that reference data yet to be uncovered.

### CODE REVIEW

File: src/xbiscuit.c [hunk 1]
```diff
@@ -0,0 +1,285 @@
+#include "common.h"
+#include "libdma.h"
+
+typedef struct {
+    /* 0x00 */ u8 unk_00[0x84];
```
Comment (ethteck) [inline src/xbiscuit.c:5]: sorry I keep forgetting to mention this, but I like to differentiate intentional u8 arrays from unknown data by using `char` as the type for the unknown space
```suggestion
    /* 0x00 */ char unk_00[0x84];
```

File: src/xbiscuit.c [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+extern s32 D_003EBCC0;
+extern s32 D_003EBCC4;
+extern s32 D_003E38C0;
+extern s32 D_003EB8C0;
+
+extern s32 D_00639A80;
+extern s32 D_00639A84;
+extern s32 D_00639A88;
+extern s32 D_00639A8C;
+XCookie D_00639A90[24];
```
Comment (ethteck) [inline src/xbiscuit.c:26]: 24 cookies, yummy

File: src/xquack.c [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+    func_001EE0B8(38, arg0, arg1, 0, 0, 0, 0, 0);
+    return D_0062D980;
+}
+
+INCLUDE_ASM(const s32, "xquack", func_001EE228);
+
+INCLUDE_ASM(const s32, "xquack", func_001EE280);
+
+void* func_001EE2D8(void* arg0, void* arg1, void* arg2) {
+    D_00639880 += 1;
```
Comment (ethteck) [inline src/xquack.c:78]:
```suggestion
    D_00639880++;
```

File: src/xquack.c [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+
+void* func_001EE478(void* arg0, void* arg1, void* arg2, s32 arg3, s32 arg4) {
+    func_001EE0B8(8, arg0, arg1, arg2, arg3, arg4, 0, 0);
+    return D_0062D980;
+}
+
+INCLUDE_ASM(const s32, "xquack", func_001EE4C8);
+
+void* func_001EE518(void* arg0, void* arg1, void* arg2) {
+    D_00639880 += 1;
```
Comment (ethteck) [inline src/xquack.c:102]: here2 (all += 1 should be ++ and all -=1 --)

File: src/xtango.c [hunk 5]
```diff
@@ -0,0 +1,339 @@
...
+} XWhiskey;
+
+u16 D_003DE5F0[12];
+u32 D_003DE630[10];
+XWhiskey* D_003ED718;
+
+INCLUDE_ASM(const s32, "xtango", func_001EF210);
+
+u32 func_001EF298() {
+    return func_001EF210() / 216000u;
```
Comment (ethteck) [inline src/xtango.c:24]: are the us necessary here? I'd prefer to omit them if possible

**Review by ethteck — CHANGES_REQUESTED** [review]
great stuff! a couple style requests based on my preferred way of stuff. happy to discuss these

File: src/xtango.c (hunk 5 above)
Comment (Yotona) [reply src/xtango.c]: Added a temporary declaration to fix it, they did specifically need to be unsigned.

File: src/xbiscuit.c (hunk 1 above)
Comment (Yotona) [reply src/xbiscuit.c]: Misunderstood your last suggestion on this, thought you meant char arrays when we figured it was string-related and u8 otherwise

**Comment by Yotona** [comment]
Tiny bit of dynamic analysis later, those 99999 functions I was talking about are definitely munny!

**Pull request merged**